libraries/standard/mqtt/cbmc/sources/mqtt_cbmc_state.c

```diff
@@ -18,13 +18,139 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
-#include <stddef.h>
 #include <stdint.h>
 #include <stdlib.h>
 #include "mqtt_cbmc_state.h"
+#include "mqtt.h"
```

Comment:
You are including this mqtt.h both here and in mqtt_cbmc_state.h.  I prefer removing it from mqtt_cbmc_state.h.  I like this style here.

Reply:
I will take it out of here and leave int mqtt_cbmc_state.h because mqtt_cbmc_state.h has types from this file in it.